tests/unit/test_mosaic_utilities.py

```diff
@@ -441,8 +441,6 @@ def test_dimensions_mapping_output_spatial(self, mock_dataset):
         lat_data_two = np.array([10, 15])
         lon_data_one = np.array([1, 2, 3])
         lon_data_two = np.array([6, 7])
-        expected_output_lat_data = np.array([-10, -5, 0, 5, 10, 15])
-        expected_output_lon_data = np.array([1, 2, 3, 4, 5, 6, 7])
```

Comment:
These were giving "local variable 'X' is assigned but never used" warnings. If/when we get spatial aggregation going, the assertions that use them will be uncommented below. So for now I just shifted these variables into the same block of commented assertions.